src/coreclr/tools/aot/ILCompiler.RyuJit/JitInterface/CorInfoImpl.RyuJit.cs

```diff
@@ -2248,14 +2248,42 @@ private void getFieldInfo(ref CORINFO_RESOLVED_TOKEN pResolvedToken, CORINFO_MET
             //       and STS::AccessCheck::CanAccess.
         }
 
+        private bool canNeverHaveInstanceOfSubclassOf(TypeDesc type)
```

Comment:
```suggestion
        private bool CanNeverHaveInstanceOfSubclassOf(TypeDesc type)
```
Nit: We seem to use regular naming convention in this file for methods that are not directly exposed on JIT/EE interface